pkg/sippyserver/pr_commenting_processor.go

```diff
@@ -555,6 +543,59 @@ func (aw *AnalysisWorker) processRiskAnalysisComment(prPendingComment models.Pul
 	log.Debugf("Comment added to pendingComments: %s/%s/%s", pendingComment.org, pendingComment.repo, pendingComment.sha)
 }
 
+func buildComment(sortedAnalysis RiskAnalysisEntryList, sha string) string {
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("Job Failure Risk Analysis for sha: %s\n\n| Job Name | Failure Risk |\n|:---|:---|\n", sha))
+
+	for _, value := range sortedAnalysis {
+		tableKey := value.Key
+
+		if value.Value.URL != "" {
+			tableKey = fmt.Sprintf("[%s](%s)", value.Key, value.Value.URL)
+		}
+
+		var riskSb strings.Builder
+		riskSb.WriteString(fmt.Sprintf("<b>%s</b>", value.Value.RiskLevel.Name))
+		for i, t := range value.Value.TestRiskAnalysis {
+			if i > 3 {
+				break
```

Comment:
This looks like our safety mechanism to not print thousands of test results in, could we drop a note in, like "+25 more..."